solutions/009_Find_the_Runner_Up_Score.md

```diff
@@ -92,3 +92,17 @@ if __name__ == '__main__':
     ordr = sorted(scores, reverse=True)
     print(ordr[1])
 ```
+
+
+
+## Solution 4
+
+```
+def remove_n(list, n):
+    new_arr = [num for num in list if num != n]
+    return new_arr
+
+arr = list(arr)
```

Comment:
Thanks for your contribution. Could you test your solution on HackerRank and make sure pass the tests first? There is a user input issue.

Reply:
```
def remove_n(list, n):
    new_arr = [num for num in list if num != n]
    return new_arr

arr = list(arr)
new_arr = remove_n(arr, max(arr))
print(max(new_arr))
```

This is the code I used, and which I believe I added to your file? It passed all the tests, you can check it out [here](https://www.hackerrank.com/challenges/find-second-maximum-number-in-a-list/submissions/code/294723294) :)

Reply:
Ah I see, the given code part was missing on my HR that's why it didn't work for me 😅 OK I'll approve your solution.